src/mixins/java/org/spongepowered/common/mixin/api/mcp/entity/player/ServerPlayerEntityMixin_API.java

```diff
@@ -448,7 +449,13 @@ public void remove() {
     @Override
     public boolean setLocation(final Vector3d position, final UUID world) {
```

Comment:
So funny enough, this is not implemented correctly in API 7...

https://github.com/SpongePowered/SpongeAPI/blob/api-8/src/main/java/org/spongepowered/api/entity/living/player/User.java#L95

It is supposed to NEVER load a world. It queries if the World is online and bombs if it isn't.